model/team_test.go

```diff
@@ -95,9 +88,8 @@ var domains = []struct {
 
 func TestValidTeamName(t *testing.T) {
 	for _, v := range domains {
-		if IsValidTeamName(v.value) != v.expected {
-			t.Errorf("expect %v as %v", v.value, v.expected)
-		}
+		actual := IsValidTeamName(v.value)
+		assert.Equal(t, actual, v.expected)
```

Comment:
i believe the `Equal` api expects `expected` first and then actual. https://godoc.org/github.com/stretchr/testify/assert#Equal. So this should be flipped

Reply:
fixed.